src/lib/plugin.js

```diff
@@ -128,6 +128,17 @@ class FiveBellsLedger extends EventEmitter2 {
     const ledgerMetadata = yield this._fetchLedgerMetadata()
     this.accountUriTemplate = ledgerMetadata.urls.account
 
+    // Set ILP prefix
+    const ledgerPrefix = ledgerMetadata.ilp_prefix
+    this.prefix = ledgerMetadata.ilp_prefix || this.fallbackPrefix
+
+    if (ledgerPrefix && this.fallbackPrefix && ledgerPrefix !== this.fallbackPrefix) {
+      throw new Error('Unexpected ledger prefix: does not match local config')
```

Comment:
Sorry for nitpicking, but if it'll throw an error if they don't match I wouldn't call it the "fallbackPrefix". That name makes it seem like it's just there in case there isn't one but it would be ignored if there is